documentation/Architecture/Overview.md

```diff
@@ -17,20 +17,20 @@ level, it exposes the following interfaces:
 
 The interfaces depicted in this diagram are
 
-1) The SDK takes as input *data sources*, such as file paths, from the *SDK driver*. The SDK may receive this input when, for example, the user asks 
+1) The SDK receives *data sources* (e.g. file paths) from the *SDK driver*. For example, the SDK may receive this input when the user asks 
```

Comment:
Would you be okay if we remove "file paths" altogether? I don't think it is that crucial to this sentence and somewhat interrupts the flow. Especially since we have an example in the next sentence

Reply:
Fine with me!